python_binding/tests/pravega_client_test.py

```diff
@@ -13,9 +13,46 @@
 import string
 import pravega_client
 from pravega_client import TxnFailedException
+from pravega_client import StreamScalingPolicy
+from pravega_client import StreamRetentionPolicy
 
 class PravegaTest(unittest.TestCase):
 
+    def test_tags(self):
```

Comment:
Probably we need to add a test case when there are no tags, then the result should be empty.

Reply:
done, improve the test.